docs/getting-started/projects.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
Can we rename the file to `projects-json.md`?

Reply:
Updated it